ipaserver/install/dogtaginstance.py

```diff
@@ -275,6 +280,72 @@ def uninstall(self):
             logger.critical("failed to uninstall %s instance %s",
                             self.subsystem, e)
 
+    def __is_newer_tomcat_version(self, default=None):
```

Comment:
I would not set a default value default=None as it would raise AttributeError if the caller forgets to specify a version.